locales/en/plugin__kubevirt-plugin.json

```diff
@@ -699,6 +699,7 @@
   "No alerts found": "No alerts found",
   "No available boot source": "No available boot source",
   "No boot source": "No boot source",
+  "No Bootable volumes found": "No Bootable volumes found",
```

Comment:
If you are using generic nouns and not k8s object names, they should follow sentence case: "No bootable volumes found"